data-model.sql

```diff
@@ -428,9 +416,6 @@ ALTER TABLE query_organization OWNER to pulse;
 
 CREATE TABLE patient_record (
 	id bigserial not null,
-	organization_patient_id varchar(1024) not null,
```

Comment:
The "organization_patient_id" should actually still be there. It's meant to hold whatever ID the organization assigns to that patient. I think it's like an alphanumeric GUID that comes back in the SOAP response for patient discovery.